### ISSUE TITLE
Mapping for math and mphantom tags are missing

### ISSUE THREAD

**Issue by fred-wang** [w3c/mathml-aam#9]
cc @asurkov 

MathML Core elements are listed here: https://w3c.github.io/mathml-core/#mathml-elements-and-attributes

It seems mapping for `<math>` and `<mphantom>` are missing.

`<mphantom>` is essentially an `<mrow>` with default style `visibility: hidden` ( https://w3c.github.io/mathml-core/#making-sub-expressions-invisible-mphantom ) so it generally does not show up in the accessibility tree. What should we do if someone reverts that CSS style? Still don't create any node in the accessibility tree? Or follow the same mapping as for `<mrow>`?

The mapping for `<math>` is defined in https://w3c.github.io/html-aam/ although it's not a HTML element. Note that the same is true for `<svg>` but mapping it still repeated in https://w3c.github.io/svg-aam/

**Comment by asurkov** [comment]
Sounds like a two separate issue with me, but

I'd say `<mphantom>` should not be mapped to the accessibility tree until default styling is applied; if the style gets overridden and the element gets visible then it should be mapped the same way as `mrow` element.

`<math>` should be moved to mathml-amm spec from html-aam apparently. Something suggests me that HTML5 used to define `<math>` and that's why it was placed into html-aam, but I might be wrong on it. Now MathML core defines `<math>` element (https://w3c.github.io/mathml-core/#the-top-level-math-element), so it should be moved to mathml-aam spec.

**Comment by fred-wang** [comment]
> Sounds like a two separate issue with me, but

Yes, I was checking the list of unmapped elements and found only two, so I thought they could go in the same issue.

> I'd say `<mphantom>` should not be mapped to the accessibility tree until default styling is applied; if the style gets overridden and the element gets visible then it should be mapped the same way as `mrow` element.

Default style is visibiliy: hidden, so I think it's not exposed by default. And treating as `mrow` and similar elements sounds good to me.

> `<math>` should be moved to mathml-amm spec from html-aam apparently. Something suggests me that HTML5 used to define `<math>` and that's why it was placed into html-aam, but I might be wrong on it. Now MathML core defines `<math>` element (https://w3c.github.io/mathml-core/#the-top-level-math-element), so it should be moved to mathml-aam spec.

I think it was in MathML3 in the past and the mathml-aam spec didn't exist. Anyway, I guess we can add it anyway (like svg-aam does) and wait for https://github.com/w3c/html-aam/issues/344
